service-base-quarkus/src/main/java/org/eclipse/hono/service/quarkus/AbstractServiceApplication.java

```diff
@@ -54,4 +67,70 @@ protected void logJvmDetails() {
                     CpuCoreSensor.availableProcessors());
         }
     }
+
+    /**
+     * Enables collection of JVM related metrics.
+     * <p>
+     * Enables collection of Memory, Thread, GC and Processor metrics.
+     */
+    protected void enableJvmMetrics() {
+        new ProcessorMetrics().bindTo(meterRegistry);
+        new JvmMemoryMetrics().bindTo(meterRegistry);
+        new JvmThreadMetrics().bindTo(meterRegistry);
+        this.jvmGcMetrics = new JvmGcMetrics();
+        jvmGcMetrics.bindTo(meterRegistry);
```

Comment:
I have experimented with the *quarkus-micrometer* extension for several days but was not able to configure it in a way that it would create the (desired) PrometheusMeterRegistry, but not create an extra HTTP server for exposing the scrape endpoint. We already expose the scrape endpoint via the HealthCheckServer so I do not see why we should spawn an additional HTTP server which basically does nothing ...
I'd rather stick with our current approach and activate the metrics that we actually want. I had left out the *ClassLoaderMetrics* and *JvmHeapPressureMetrics* as I do not believe that anybody is (currently) interested in them, but I might be mistaken.

Reply:
Ah, ok. Yes, makes sense to avoid creating another HTTP server here.